privacy.html

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+---
+
+<div class="row justify-content-center">
+  <div class="col-10">
+    <h1>Privacy Policy for Two Factor Auth</h1>
+    <p>At Two Factor Auth, accessible from https://twofactorauth.org/, one of our main priorities is the privacy of our
+      visitors. This Privacy Policy document contains types of information that is collected and recorded by Two Factor
+      Auth and how we use it.</p>
+    <h2>General Data Protection Regulation (GDPR)</h2>
+    <p>We are a Data Controller of your information.</p>
+    <p>Two Factor Auth legal basis for collecting and using the personal information described in this Privacy Policy
+      depends on the Personal Information we collect and the specific context in which we collect the information:</p>
+    <ul>
+      <li>Two Factor Auth needs to perform a contract with you</li>
+      <li>You have given Two Factor Auth permission to do so</li>
+      <li>Processing your personal information is in Two Factor Auth legitimate interests</li>
+      <li>Two Factor Auth needs to comply with the law</li>
+    </ul>
+    <p>Two Factor Auth will retain your personal information only for as long as is necessary for the purposes set out
+      in
+      this Privacy Policy. We will retain and use your information to the extent necessary to comply with our legal
+      obligations, resolve disputes, and enforce our policies.</p>
+    <p>If you are a resident of the European Economic Area (EEA), you have certain data protection rights. If you wish
+      to
+      be informed what Personal Information we hold about you and if you want it to be removed from our systems, please
+      contact us.</p>
+    <p>In certain circumstances, you have the following data protection rights:</p>
+    <ul>
+      <li>The right to access, update or to delete the information we have on you.</li>
+      <li>The right of rectification.</li>
+      <li>The right to object.</li>
+      <li>The right of restriction.</li>
+      <li>The right to data portability</li>
+      <li>The right to withdraw consent</li>
+    </ul>
+    <h2>Data Access</h2>
+    <p>The only parties that have access to the data include Josh Davis (Seattle, WA) as well
+      as Google LLC (1600 Amphitheater Parkway, Mountain View, CA 94043, USA)</p>
+    <h2>Google Analytics</h2>
+    <p>The services contained in this section enable the Owner to monitor and analyze web traffic and can be used to keep track of User behavior. </p>
+    <p>Google Analytics is a service that analyzes web traffic and is provided by Google LLC (“Google”). The data is collected and used to prepare
+      reports on user activity and shares it with Google's services. The data collected includes but not limited to: referrer, page visit data
+      such as time of visit, duration of visit, web browser and OS details. Data is anonymized.</p>
```

Comment:
```suggestion
      such as time of visit, duration of visit, web browser and OS details. Data is pseudo-anonymized.</p>
```
Anonomized data would mean it can't be correlated to a sender.